apps/perf-test/src/scenarios/FluentProviderWithTheme.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+
+const LayoutShift: React.FunctionComponent = ({ children }) => {
+  // eslint-disable-next-line no-restricted-properties
+  React.useLayoutEffect(() => {
+    if (document.body) {
+      document.body.offsetWidth;
```

Comment:
Why do you just reference a read only property ? perhaps a comment there would be nice since it is a mystery to me

Reply:
This is to force reflow to be able to measure styles rendering perf.
https://gist.github.com/paulirish/5d52fb081b3570c81e3a